### ISSUE TITLE
Support browser zooming up to 200%

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#396]
One a11y check I tota11y forgot about until now is the zooming of the page in browser... :disappointed: 
[WCAG recs](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html) state that the success criterion is satisfied when user is able to zoom in up to 200% and still retain proper access to all the elements in the page. Here is what we stand if I test the Kolibri content zoomed to 200% on the [most prevalent laptop monitor size today](https://www.w3counter.com/globalstats.php) (1366x768px):

**Device Admin cannot reach the user/log out menu item**
![kolibri - chromium_638](https://cloud.githubusercontent.com/assets/1457929/17671679/931871f2-6318-11e6-9374-1510f6d9e487.png)

**Add New Account modal**
![kolibri - mozilla firefox_636](https://cloud.githubusercontent.com/assets/1457929/17671726/c734948e-6318-11e6-8e9d-e0109db38729.png)

**Edit User Account modal**
![kolibri - mozilla firefox_637](https://cloud.githubusercontent.com/assets/1457929/17671745/e0c1cc28-6318-11e6-9841-0607deb4a12b.png)

**User content (document) Learn page - title remains cut out (invisible)**
![kolibri - chromium_640](https://cloud.githubusercontent.com/assets/1457929/17671827/44b9712c-6319-11e6-9fa7-5852e4115253.png)

Maybe another reason to reconsider the placement of the H1? @indirectlylit @whitzhu @66eli77

**Comment by indirectlylit** [comment]
Let's try to break this up and handle each case individually

* created a new task to update the nav-bar design: https://trello.com/c/o4cFBP2A/692-nav-bar-handle-small-screens-and-large-numbers-of-items
* modals now have scroll bars and should be able to handle small windows
* I'm not reproducing the header issue, but might be related to https://trello.com/c/d3209HUZ/664-h1-cut-off-on-landing-page-fresh-dev-repo or https://github.com/learningequality/kolibri/issues/646 ?
